src/Bindings/ManualBindings.cpp

```diff
@@ -3684,6 +3684,48 @@ static int tolua_cEntity_GetSpeed(lua_State * tolua_S)
 
 
 
+static int tolua_cColor_Set(lua_State * tolua_S)
+{
+	cLuaState L(tolua_S);
+
+	if (!L.CheckParamNumber(2, 4) || !L.CheckParamSelf("cColor"))
+	{
+		return 0;
+	}
+	cColor * self = reinterpret_cast<cColor *>(tolua_tousertype(tolua_S, 1, nullptr));
```

Comment:
It would still be nice to have the `Get` method that returns all three components; ToLua cannot export such a function, so ManualBindings for it should be done.

Alright, keep the enum untyped for now. @peterbell10 would know more about those typed enums.